### PR TITLE
Add Settings Screen

### LINKED ISSUE #53: [2pts] Settings Screen
**Fill in the blanks**:
As a student, I would like to customize the app, so that I get the most useful data quicker.

**Describe the solution you'd like:**
- [x] Add a title to identify the page
- [x] Create button (one button per line) for 'food preferences' to add functionality to later
- [x] Create button (one button per line) for 'Quack mission statement' to add functionality to later
- [x] Have a 'back' button to add functionality to later
- [x] Contains Nav Bar to add functionality to later

**Definition of Done - Check as you go!**
- [x] Tasks have been implemented
- [x] Documented
- [x] Passing Tests for each task
- [x] Pull Request made
- [ ] Code Reviewed

### PR DESCRIPTION
**Pull request by RafaelPiloto10**
This pull request adds the Settings screen which contains list tiles for each item outlined in the paper prototype.
The buttons route to an empty page with the title bar

Screenshot:
Screen Shot 2021-10-16 at 2 13 48 PM

### CODE REVIEW

File: quack_app/lib/screens/settings/settings_screen.dart
Comment (DDVD233) [inline quack_app/lib/screens/settings/settings_screen.dart:55]: Probably not an issue, but does this initialize every sub-setting view while loading this settings view?

File: quack_app/lib/components/navbar.dart
Comment (DDVD233) [inline quack_app/lib/components/navbar.dart:25]: I feel like settings should be the last one (the rightmost one)

**Review by DDVD233 — APPROVED** [review]

File: quack_app/lib/screens/settings/settings_screen.dart
Comment (RafaelPiloto10) [reply quack_app/lib/screens/settings/settings_screen.dart:55]: Potentially? But I guess the same could be said about how the Navbar works?

File: quack_app/lib/components/navbar.dart
Comment (RafaelPiloto10) [reply quack_app/lib/components/navbar.dart:25]: @MimiOlayeye, what do you think?

**Comment by oreorz** [comment]
Noticed 'prefrences' spelling on a couple of files, should be 'preferences'.

File: quack_app/lib/screens/settings/settings_screen.dart
Comment (brendacano) [inline quack_app/lib/screens/settings/settings_screen.dart]: fix all instances of "prefrences" and change to "preferences"

File: quack_app/lib/components/navbar.dart
Comment (brendacano) [reply quack_app/lib/components/navbar.dart:25]: Hmm, that makes sense. IMO I think it will feel the same on either side because most apps don't have a settings option at the bottom and instead have it like in the top corner. Basically what I'm saying is that it's fine on either side

**Review by brendacano — CHANGES_REQUESTED** [review]
please fix the spelling

**Review by brendacano — CHANGES_REQUESTED** [review]
**i think the names of these files/pngs should also be fixed:**

account_prefrences_screen_test.dart

food_prefrences_screen_test.dart

settings_account_prefrences_screen.png

settings_food_prefrences_screen_view.png

settings_notifications_prefrences_screen_view.png

**Review by Foxworth22 — APPROVED** [review]
Looks like a lot of additions, good work!

**Comment by oreorz** [comment]
Looks like you have a merge conflict. Apart from that, everything looks good. Will approve once merge conflicts fixed

**Review by brendacano — APPROVED** [review]
looks good

**Pull request merged**
